### PR TITLE
Vendor in latest Buildah

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

Title says it all.

### CODE REVIEW

**Comment by mheon** [comment]
Changes LGTM.

While parsing through the diff, I noticed y'all are using libnetwork's resolvconf package in Buildah. We ended up copying it into Libpod because trying to vendor it was bringing in somewhere around 50k lines of code; if you've managed to import it for Buildah without doing that, we should delete our local copy and swap to the vendored one once this merges.

File: vendor.conf [hunk 1]
```diff
@@ -112,3 +112,4 @@ gopkg.in/tomb.v1 v1
 github.com/spf13/cobra v0.0.3
 github.com/inconshreveable/mousetrap v1.0.0
 gopkg.in/fsnotify.v1 v1.4.7
+github.com/ishidawataru/sctp 07191f837fedd2f13d1ec7b5f885f0f3ec54b1cb
```
Comment (TomSweeneyRedHat) [inline vendor.conf]: I'd to add this line and bump libnetwork due to the changes @QiWang19  made for https://github.com/containers/buildah/pull/1491

Maybe I'm not remembering right, but didn't thought the Buildah vendor would pull that in automagically.   Any who when I tried building libpod it failed until I added them.

**Comment by rhatdan** [comment]
Let's go back and copy it into buildah and then revendor it into libpod, removing that vendor.

**Comment by mheon** [comment]
No, it's worth using the vendored one if I aren't including massive amounts
of unnecessary code, and that seems to be the case. I'd just as soon merge
this and delete the local implementation we have.

**Comment by TomSweeneyRedHat** [comment]
bot, retest this please

**Comment by TomSweeneyRedHat** [comment]
Had to change a few tests  due to the way layers now works.
Prior: 
```
# podman build -t foobar.com/before:latest --layers=false -f ~/Dockerfile .
STEP 1: FROM docker.io/library/alpine:latest
Getting image source signatures
Copying blob bdf0201b3a05 done
Copying config cdf98d1859 done
Writing manifest to image destination
Storing signatures
STEP 2: COMMIT containers-storage:[overlay@/var/lib/containers/storage+/var/run/containers/storage:overlay.mountopt=nodev]foobar.com/before:latest
Getting image source signatures
Copying blob a464c54f93a9 skipped: already exists
Copying blob 1dbcab28ce46 done
Copying config 559fd5095f done
Writing manifest to image destination
Storing signatures
--> 559fd5095ff8fe0d03e4dd4789623681e67e0513f2abd525bd08432dd634f6c6
STEP 3: COMMIT foobar.com/before:latest@sha256:7999747fab75d4e9ab42be4cff7d5ea666b58863b53fefedcf471625b54f1a7e

# podman images
REPOSITORY                 TAG      IMAGE ID       CREATED         SIZE
foobar.com/before          latest   559fd5095ff8   4 seconds ago   5.8 MB
docker.io/library/alpine   latest   cdf98d1859c1   6 days ago      5.8 MB

```
New:
```
# podman build -t foobar.com/before:latest --layers=false -f ~/Dockerfile .
STEP 1: FROM docker.io/library/alpine:latest
Getting image source signatures
Copying blob bdf0201b3a05 done
Copying config cdf98d1859 done
Writing manifest to image destination
Storing signatures
STEP 2: COMMIT foobar.com/before:latest
--> cdf98d1859c1beb33ec70507249d34bacf888d59c24df3204057f9a6c758dddb

# podman images
REPOSITORY                 TAG      IMAGE ID       CREATED      SIZE
docker.io/library/alpine   latest   cdf98d1859c1   6 days ago   5.8 MB
foobar.com/before          latest   cdf98d1859c1   6 days ago   5.8 MB
```

Note the image ids are now the same.

@nalind, please verify this is expected, I believe it is.

**Comment by nalind** [comment]
@TomSweeneyRedHat yes, when there are no instructions we don't create a new layer.  This is going to require some changes to code that assumes that every image has more layers than an image that it's based on, which has not always been the case for images built using `docker build`.

**Comment by TomSweeneyRedHat** [comment]
Happy Green Test buttons

**Pull request merged**